tests/components/onewire/test_entity_owserver.py

```diff
@@ -111,15 +111,15 @@
                 "unique_id": "/1D.111111111111/counter.A",
                 "injected_value": b"    251123",
                 "result": "251123",
-                "unit": "count",
+                "unit": None,
```

Comment:
Why is the test changing?

Reply:
You are right - this should be a separate PR

Reply:
Removed in 4bd507e